### PR TITLE
Extra POI Vengoth

### PR DESCRIPTION
**Pull request by AndriesDraux**

### CODE REVIEW

File: extra/points-of-interest/markers.json [hunk 1]
```diff
@@ -4899,13 +4920,6 @@
 		"y": 32821,
 		"z": 7
 	},
-	{
-		"description": "",
-		"icon": "red down",
-		"x": 32201,
-		"y": 32948,
-		"z": 7
-	},
```
Comment (mathiasbynens) [inline extra/points-of-interest/markers.json:4908]: Is this removal intentional?

Comment (AndriesDraux) [reply extra/points-of-interest/markers.json:4908]: It isn't, however not sure where it came from as I did not remove any markers
